data/packages/com.rhinox.open.grpl2it.yml

```diff
@@ -8,15 +8,14 @@ licenseName: Apache-2.0
 topics:
   - frameworks
   - input-management
-  - level-design
   - physics
   - utilities
 hunter: GlennQuintyn
 gitTagPrefix: ''
 gitTagIgnore: ''
 minVersion: ''
 image: ''
-readme: 'main:README.md'
+readme: 'IT_README.md'
```

Comment:
The readme is parsed from GitHub instead of the package root, please change it to `main:/Assets/GrappleIT/IT_README.md` (`main:` is the branch)